OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineActivity.java

```diff
@@ -41,14 +37,11 @@ protected void onCreate(Bundle savedInstanceState) {
         }
     }
 
-    public void onResume(){
+    public void onResume() {
         super.onResume();
 
         if (isOnCourseOutline) {
             setTitle(courseData.getCourse().getName());
-            if (!isVideoMode) {
```

Comment:
⚠️
Decide either it require to delete all Video mode course outline functionality, if we are going to remove this feature make sure it's removed cleanly because its still using in the code of this class.